includes/class-integrate-convertkit-wpforms-api.php

```diff
@@ -44,10 +44,10 @@ public function __construct( $api_key = false, $api_secret = false, $debug = fal
 		$this->api_key        = $api_key;
 		$this->api_secret     = $api_secret;
 		$this->debug          = $debug;
-		$this->plugin_name    = ( defined( 'CKWC_PLUGIN_NAME' ) ? CKWC_PLUGIN_NAME : false );
-		$this->plugin_path    = ( defined( 'CKWC_PLUGIN_PATH' ) ? CKWC_PLUGIN_PATH : false );
-		$this->plugin_url     = ( defined( 'CKWC_PLUGIN_URL' ) ? CKWC_PLUGIN_URL : false );
-		$this->plugin_version = ( defined( 'CKWC_PLUGIN_VERSION' ) ? CKWC_PLUGIN_VERSION : false );
+		$this->plugin_name    = ( defined( 'INTEGRATE_CONVERTKIT_WPFORMS_NAME' ) ? INTEGRATE_CONVERTKIT_WPFORMS_NAME : false );
```

Comment:
Where are these constants coming from?

Reply:
They're defined in the main Plugin file:
https://github.com/ConvertKit/convertkit-wpforms/blob/fix-user-agent-constants/integrate-convertkit-wpforms.php#L35-L39